### PR TITLE
🌐 Add Spanish translation for `docs/es/docs/environment-variables.md`

### PR DESCRIPTION
**Pull request by tintayadev**

### CODE REVIEW

File: docs/es/docs/environment-variables.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+Hello Wade Wilson
+```
+
+</div>
+
+////
+
+## Lee Variables de Entorno en Python
+
+También puedes crear variables de entorno **por fuera** de Python, en la terminal (o por cualquier otra método), y luego **leerlas en Python**.
```
Comment (JavierSanchezCastro) [inline docs/es/docs/environment-variables.md:55]:
```suggestion
También puedes crear variables de entorno **por fuera** de Python, en la terminal (o por cualquier otro método), y luego **leerlas en Python**.
```

**Review by JavierSanchezCastro — CHANGES_REQUESTED** [review]

File: docs/es/docs/environment-variables.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
+# Variables de Entorno
+
+/// tip
```
Comment (JavierSanchezCastro) [inline docs/es/docs/environment-variables.md:3]:
```suggestion
/// tip | Consejo
```

File: docs/es/docs/environment-variables.md [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+Por ejemplo, podrías tener un archivo `main.py` con:
+
+```Python hl_lines="3"
+import os
+
+name = os.getenv("MY_NAME", "World")
+print(f"Hello {name} from Python")
+```
+
+/// tip
```
Comment (JavierSanchezCastro) [inline docs/es/docs/environment-variables.md:66]:
```suggestion
/// tip | Consejo
```

File: docs/es/docs/environment-variables.md [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+
+// La variable de entorno ya no existe después
+$ python main.py
+
+Hello World from Python
+```
+
+</div>
+
+/// tip
```
Comment (JavierSanchezCastro) [inline docs/es/docs/environment-variables.md:154]:
```suggestion
/// tip | Consejo
```

**Review by JavierSanchezCastro — CHANGES_REQUESTED** [review]

**Comment by tintayadev** [comment]
Hey @JavierSanchezCastro 
Thanks so much for your review, I miss those typos.

**Review by JavierSanchezCastro — APPROVED** [review]

**Comment by tiangolo** [comment]
This was covered in another PR, so I'll close this one now, thanks for the effort! :cake: :coffee:

**Pull request closed without merging**